files/default/kibana-4.x-default

```diff
@@ -1,6 +0,0 @@
-user="kibana"
```

Comment:
this file isn't necessary anymore since we switched to installing kibana from a .deb package on a recent release